packages/core/src/Stores/client-store.js

```diff
@@ -1633,6 +1639,9 @@ export default class ClientStore extends BaseStore {
             this.resetMt5AccountListPopulation();
         }
         this.responseWebsiteStatus(await WS.wait('website_status'));
+        if (this.is_tnc_status_updated?.landing_company_short === 0) {
```

Comment:
can you elaborate what is this for?

Reply:
We check for the tnc_status for current landing company and we see if its value is 0. As 0 indicates, the client has not updated. The reason I did not check `!this.is_tnc_status_updated?.landing_company_short === 0` was because it becomes true for undefined. Else we need to add additional checks for landing_company if its svg/malta.